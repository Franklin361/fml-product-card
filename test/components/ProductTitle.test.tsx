import React from 'react';
import render from 'react-test-renderer';
import { ProductTitle, ProductCard } from '../../src/components';

import { product } from '../data/products';

describe('Product Title', () => {
    
    test('Debe de Mostrar el componente con el titulo Personalizado', () => {
        const wrapper = render.create(
            <ProductTitle title="Example Title" />
        )

        expect(wrapper.toJSON()).toMatchSnapshot();
    })

    test('Debe de mostrar el componente con el nombre del producto', () => {
        
        const wrapper = render.create(
            <ProductCard product={product}>
                {
                    () => (
                        <ProductTitle/>
                    )
                }
            </ProductCard>
        )

        expect(wrapper.toJSON()).toMatchSnapshot();

    })
    

})
