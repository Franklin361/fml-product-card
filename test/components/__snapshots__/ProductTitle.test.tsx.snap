// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Title Debe de Mostrar el componente con el titulo Personalizado 1`] = `
<span
  className="productDescription undefined"
>
  Example Title
</span>
`;

exports[`Product Title Debe de mostrar el componente con el nombre del producto 1`] = `
<div
  className="productCard undefined"
>
  <span
    className="productDescription undefined"
  >
    Producto 1
  </span>
</div>
`;
